.ci/jobs/bump-opbeans-node-mbp.yml

```diff
@@ -1,41 +0,0 @@
----
-- job:
```

Comment:
@elastic/observablt-ci Will this automatically delete the Jenkins job? Or is any follow up action required?

Reply:
no, the job is not deleted from Jenkins you have to login and delete it